precompiles/vesting/src/contracts/Vesting.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity >=0.8.3;
+
+/// @title Pallet Vesting Interface (pallet code here: https://github.com/paritytech/polkadot-sdk/tree/master/substrate/frame/vesting)
+/// @author LAOS Team
+/// @notice This interface allows Solidity contracts to interact with pallet-vesting
+/// @custom:address 0x0000000000000000000000000000000000000407
+interface Vesting {
+
+    /// @dev Defines the vesting info
+    /// @dev `locked` is the amount of locked tokens
+    /// @dev `perBlock` is the amount of unlocking tokens per block
+    /// @dev `startingBlock` is the block number at which the tokens start unlocking
+    struct VestingInfo {
+        uint256 locked;
+        uint256 perBlock;
+        uint256 startingBlock;
+    }
+
+    /// @notice Returns the vesting info of an account
+    /// @param _target The address of the account the vesting data should be returned for
+    /// @return The vesting info of the account
```

Comment:
I suggest making it a bit more explicit, perhaps, "The Vesting info of the provided account, as an array of Vesting Info structs". As you like

Reply:
fixed